app/src/pages/inside/dashboardItemPage/dashboardItemPage.scss

```diff
@@ -0,0 +1,45 @@
+.dashboard-item {
+  background: white;
+
+  .gadget-view {
+    border: 1px solid grey;
```

Comment:
Use color variables to define colors in styles

Reply:
done